Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Data.cs

```diff
@@ -52,5 +52,86 @@ public enum GridResizeBehavior
             /// </summary>
             PreviousAndNext
         }
+
```

Comment:
Because I need to add a default value for the enum, this default value represents the default behavior I created at the beginning (cursor changes with the resize direction).
what do you think ?